gpu-operator/gpu-operator-kubevirt.rst

```diff
@@ -410,15 +412,27 @@ At runtime, adminstrators then point the vGPU Device Manager at one of these con
 The configuration file is created as a ConfigMap, and is shared across all worker nodes.
 At runtime, a node label, ``nvidia.com/vgpu.config``, can be used to decide which of these configurations to actually apply to a node at any given time.
 If the node is not labeled, then the ``default`` configuration will be used.
-For more information on this component and how it is configured, refer to the project `README <https://github.com/NVIDIA/vgpu-device-manager>`_.
+For more information on this component and how it is configured, refer to the `NVIDIA vGPU Device Manager README <https://github.com/NVIDIA/vgpu-device-manager>`_.
 
-By default, the GPU Operator deploys a ConfigMap for the vGPU Device Manager, containing named configurations for all `vGPU types <https://docs.nvidia.com/grid/latest/grid-vgpu-user-guide/index.html#supported-gpus-grid-vgpu>`_ supported by NVIDIA vGPU.
+By default, the GPU Operator deploys a ConfigMap for the vGPU Device Manager, containing named configurations for all `vGPU types supported by NVIDIA vGPU <https://docs.nvidia.com/grid/latest/grid-vgpu-user-guide/index.html#supported-gpus-grid-vgpu>`_.
 Users can select a specific configuration for a worker node by applying the ``nvidia.com/vgpu.config`` node label.
-For example, labeling a node with ``nvidia.com/vgpu.config=A10-8Q`` would create 3 vGPU devices of type **A10-8Q** on all **A10** GPUs on the node (note: 3 is the maximum number of **A10-8Q** devices that can be created per GPU).
+For example, labeling a node with ``nvidia.com/vgpu.config=A10-8Q`` would create three vGPU devices of type **A10-8Q** on all **A10** GPUs on the node. Note that three is the maximum number of **A10-8Q** devices that can be created per GPU.
 If the node is not labeled, the ``default`` configuration will be applied.
-The ``default`` configuration will create Q-series vGPU devices on all GPUs, where the amount of framebuffer memory per vGPU device
-is half the total GPU memory.
-For example, the ``default`` configuration will create two **A10-12Q** devices on all **A10** GPUs, two **V100-8Q** devices  on all **V100** GPUs, and two **T4-8Q** devices on all **T4** GPUs.
+The ``default`` configuration will create Q-series vGPU devices on all GPUs, where the amount of framebuffer memory per vGPU device is half the total GPU memory.
+For example, the ``default`` configuration will create two **A10-12Q** devices on all **A10** GPUs, two **V100-8Q** devices on all **V100** GPUs, and two **T4-8Q** devices on all **T4** GPUs.
+
+You can also create different vGPU Q profiles on same GPU using vGPU Device Manager configuration.
+For example, you can create a **A10-4Q** and a **A10-8Q** device on same GPU by creating a vGPU Device Manager configuration with the following content:
```

Comment:
Did you mean L4-4Q / L4-8Q? The text here does not match the configuration snippet.

Reply:
Do we want to be consistent throughout this text and stick to the A10 GPU? We have examples below that use different A10 profiles.

What about updating this example to:
```
   version: v1
   vgpu-configs:
      custom-A10-config:
         - devices: all
            vgpu-devices:
            "A10-4Q": 3
            "A10-6Q": 2
```

The A10 has 24GB of framebuffer memory, so we can fit three A10-4Q profiles and two A10-6Q profiles on a single device.

Reply:
Good catch. Sorry, I forgot to update the text accordingly. I've changed the example config to your suggestion.